superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/buildQuery.ts

```diff
@@ -70,9 +80,13 @@ export default function buildQuery(formData: QueryFormData) {
           rollingWindowOperator(fd, queryObject),
           timeCompareOperator(fd, queryObject),
           resampleOperator(fd, queryObject),
-          renameOperator(fd, queryObject),
+          renameOperator(fd, {
+            ...queryObject,
+            columns: fd.groupby,
+            ...{ is_timeseries },
```

Comment:
Is this necessary? Can't we just?
```suggestion
            is_timeseries,
```

Reply:
![image](https://user-images.githubusercontent.com/33317356/169081290-117219dd-c974-42ea-80e8-e39cab1490d1.png)